src/locales/en/translation.json

```diff
@@ -6,6 +6,9 @@
   "averageTransactionSize": {
     "header": "Average Transaction Size"
   },
+  "block": {
+    "title": "Latest Block Transactions"
```

Comment:
I would remove "Latest" from this wording

Reply:
Based on Don's feedback we will show "Transactions" title for this section